### ISSUE TITLE
Conversation API context is not deleted

### ISSUE THREAD

**Issue by petrfejfar** [watson-developer-cloud/node-red-node-watson#176]
Hello,

how to replicate:
1. Start new conversation only with `msg.payload = "anything"` (btw, you can't send empty string, because you get error) and store `context` object
2. Send second message with `context` from previous call set via `msg.params.context`
3. Start new conversation similar as in first step
4. You don't get welcome message as you should, but you continue in conversation from step 2 -> bug

Bug is located in file `./services/conversation/v1.js`

Context is set in function verifyInputs:

```
// option context in JSON format
if (msg.params && msg.params.context) {
   node.context = msg.params.context;
}
```

It will survive if you don't set `msg.params.context = {}` explicitly.

**Comment by danielchabr** [comment]
+1

**Comment by chughts** [comment]
Fixed in release 0.4.10
